2d/sloping_fault/setprob.f90

```diff
@@ -0,0 +1,39 @@
+!   ==================
+    subroutine setprob
+!   ==================
+
+    implicit none
+
+    character*12 fname
+    integer iunit
+
+    REAL (kind=8) :: center(2), theta, xcb(2), mindepth
+    common /fault/  center, theta, xcb, mindepth
+
+    real (kind=8) :: width
+!
+!
+      iunit = 7
+      fname = 'setprob.data'
+!     # open the unit with new routine from Clawpack 4.4 to skip over
+!     # comment lines starting with #:
+      call opendatafile(iunit, fname)
+
+
+!
+      read(7,*) width ! domain_depth
+      read(7,*) width ! domain width
+      read(7,*) center(1)
+      read(7,*) width
```

Comment:
It's just a way to read past the domain_depth and domain_width lines in setprob.data.  There is probably a more slick/readable way to do this (by maybe reading those into a "temp" variable that is never used again).  Let me know if you think there's a better way.